### DISCUSSION TITLE
Keybind(s)

### DISCUSSION THREAD

**Discussion by cobaltcoyote** [EccentricVamp/EccentricTome#39] [Ideas]
Having an option to open the tome via keybind would 1) free up a hotbar slot and 2) let players equip it as a curio (with config editing/datapack) to always have access.

**Comment by EccentricVamp** [comment]
I investigated this a while ago. At that time, Minecraft and Forge were in the middle of changing how key bindings work internally. So, I held off on implementing it.

Hopefully things have calmed down now. I'll take another look soon. Thanks for the suggestion!
